extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/MessageBundleProcessor.java

```diff
@@ -315,6 +315,30 @@ List<MessageBundleBuildItem> processBundles(BeanArchiveIndexBuildItem beanArchiv
         return bundles;
     }
 
+    private static boolean bundleNameMatchesFileName(String fileName, String name) {
```

Comment:
It would be nice to have a plain junit test for this static method ;-). I.e. make the method package-private and add `MessageBundleProcessorTest`, something like we do in `io.quarkus.qute.deployment.QuteProcessorTest`.